addons/addon-base-raas/packages/base-raas-services/lib/schema/register-study.json

```diff
@@ -11,15 +11,17 @@
     },
     "name": {
       "type": "string",
-      "maxLength": 2048
+      "maxLength": 300,
+      "pattern": "^([A-Za-z0-9-_ ]+)$"
```

Comment:
Do we want to handle cases where study names can contain umlauts and possibly other characters.